sklearn/linear_model/tests/test_sparse_coordinate_descent.py

```diff
@@ -291,7 +291,7 @@ def test_sparse_dense_equality(
     else:
         sw = None
     Xs = csc_container(X)
-    params = {"fit_intercept": fit_intercept}
+    params = {"fit_intercept": fit_intercept, "tol": 1e-6}
```

Comment:
Shouldn't we update `sparse_enet_coordinate_descent` as part of this PR instead of changing this test?

Reply:
No, see answer below.